lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -42,24 +42,24 @@
 }
 
 .forum-nav-browse {
-    box-sizing: border-box;
-    width: 100%;
-    vertical-align: middle;
-    padding: 11px;
-    border: 0;
-    border-radius: 0;
-    background: transparent;
-    box-shadow: none;
+  box-sizing: border-box;
```

Comment:
Looks like you indented this from 4 to 2 - would rather see four ([as per styleguide](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#syntax--formatting)) but understand the change if it's for internal consistency - looks like it's been historically mixed in this file.